pegasus/sites.v3/code.org/public/educate/resources/videos.md.erb

```diff
@@ -23,14 +23,7 @@ We are building a growing library of educational videos available for re-use by
 
 <a name="how-ai-works"></a>
 <h2>How AI Works</h2>
-  <p style="margin-bottom: 25px">With an introduction by Microsoft CEO Satya Nadella, this series of short videos will introduce you to how artificial intelligence works and why it matters. Learn about neural networks, or how AI learns, and delve into issues like algorithmic bias and the ethics of AI decision-making.</p>
-
-
-  <div class="col-50" style="float: left; padding: 10px;">
-    <% facebook = {:u=>'https://youtu.be/Ok-xpKjKp2g'}
-    twitter = {:url=>'https://youtu.be/Ok-xpKjKp2g', :related=>'codeorg', :text=>'Introducing How AI Works @codeorg'} %>
-    <%=view :display_video_thumbnail, id: "introducing_ai", video_code: "Ok-xpKjKp2g", caption: "Introducing How AI Works", play_button: 'center', letterbox: 'false', facebook: facebook, twitter: twitter %>
-  </div>
+  <p style="margin-bottom: 25px">This series of short videos will introduce you to how artificial intelligence works and why it matters. Learn about neural networks, or how AI learns, and delve into issues like algorithmic bias and the ethics of AI decision-making.</p>
```

Comment:
Not blocking, as it wasn't translated earlier, but is there a plan for translating these?

Reply:
We will be doing an overhaul of the page in the next couple months so I think we leave it for now

Reply:
Makes sense, thanks!